rclcpp/include/rclcpp/publisher.hpp

```diff
@@ -195,6 +202,19 @@ class Publisher : public PublisherBase
     }
   }
 
+  void
+  do_serialized_publish(const rcl_serialized_message_t * serialized_msg)
```

Comment:
This could have been `const rcl_serializated_message_t &`, but we can do that in a follow up change. Given that it's an internally used protected, I think it could be updated after API freeze.